unittests/FEXLinuxTests/tests/signal/Syscall_state.64.cpp

```diff
@@ -0,0 +1,174 @@
+#include <catch2/catch.hpp>
+#include <csetjmp>
+#include <unistd.h>
+#include <sys/syscall.h>
+#include <signal.h>
+#include <string.h>
+#include <sys/wait.h>
+
+struct CPUState {
+  uint64_t Registers[16];
+
+  uint64_t eflags;
+};
+
+CPUState CapturedState{};
+extern "C" uint64_t HLT_INST;
+
+const void *HaltLocation = &HLT_INST;
```

Comment:
Looks like you meant to make this pointer constant, but it's actually a pointer to constant (untyped) data.

```suggestion
void * const HaltLocation = &HLT_INST;
```

Yes, I'm just as annoyed by this 🙃